src/SprykerEco/Yves/FactFinderWebComponents/Controller/IndexController.php

```diff
@@ -0,0 +1,85 @@
+<?php
+
+/**
+ * MIT License
+ * For full license information, please view the LICENSE file that was distributed with this source code.
+ */
+
+namespace SprykerEco\Yves\FactFinderWebComponents\Controller;
+
+use Spryker\Yves\Kernel\Controller\AbstractController;
+use Spryker\Yves\Kernel\View\View;
+
+/**
+ * @method \SprykerEco\Yves\FactFinderWebComponents\FactFinderWebComponentsConfig getConfig()
+ */
+/**
+ * @method \SprykerEco\Yves\FactFinderWebComponents\FactFinderWebComponentsConfig getConfig()
+ * @method \SprykerEco\Yves\FactFinderWebComponents\FactFinderWebComponentsFactory getFactory()
+ */
+class IndexController extends AbstractController
+{
+    /**
+     * @return \Spryker\Yves\Kernel\View\View
+     */
+    public function indexAction(): View
+    {
+        $twigConfigTransformer = $this->getFactory()->createWebComponentConfigToTwigConfigTransformer();
+        $webComponentsConfig = $this->getFactory()->createWebComponentsConfig();
+
+        return $this->view(
+            [
```

Comment:
The common practice is move data preparation to the `execute*()` method. E.g. `executeIndexAction()`